lib/mongo/operation/get_more/result.rb

```diff
@@ -52,7 +52,7 @@ def cursor_document
         end
 
         def first_document
-          @first_document ||= reply.documents[0]
+          @first_document ||= reply && reply.documents[0]
```

Comment:
Can this be now returned to unconditionally reading `reply`?

Reply:
Done